mobile/mobile WatchKit Extension/MotionManager.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  MotionManager.swift
+//  mobile WatchKit Extension
+//
+//  Created by Collin Prather on 4/7/20.
+//  Copyright © 2020 Collin Prather. All rights reserved.
+//
+
+import CoreMotion
```

Comment:
CoreMotion kept coming up in my search for storing data locally on the phone. I wasn't convinced that it was the way to go, but it's great to see you got it working for you!

Reply:
Hmm, well I'm using CoreMotion more for accessing the sensor data than for storing data locally on the phone, but I'm interested to hear more of your thoughts! I'm definitely not convinced this is the optimal way to do things -> 100% open to new options!